runtime/src/main/java/org/corfudb/protocols/service/CorfuProtocolMessage.java

```diff
@@ -26,6 +27,16 @@
     // Prevent class from being instantiated
     private CorfuProtocolMessage() {}
 
+    /**
+     * This type map provides clients with the ability to convert
+     * between representations of PriorityLevel.
+     */
+    public static final EnumMap<org.corfudb.protocols.wireprotocol.PriorityLevel, PriorityLevel> priorityTypeMap =
+            new EnumMap<>(ImmutableMap.of(
+                    org.corfudb.protocols.wireprotocol.PriorityLevel.NORMAL, PriorityLevel.NORMAL,
```

Comment:
please import org.corfudb.protocols.wireprotocol.PriorityLevel

Reply:
Note: this causes a conflict with `org.corfudb.runtime.proto.service.CorfuMessage.PriorityLevel`. I can instead import `org.corfudb.runtime.proto.service.CorfuMessage` and use `CorfuMessage.PriorityLevel.Normal` on the other end.

Reply:
Done.